<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sweet Tea</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="ControlPanel">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="LaunchUpdatePanel">
        <item>
         <widget class="QPushButton" name="LaunchButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Launch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ValidateButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Validate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OptionsButton">
          <property name="text">
           <string>Options</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ScreenshotButton">
          <property name="text">
           <string>Screenshots</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="UpdateProgress">
        <property name="value">
         <number>0</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
